packages/style-engine/class-wp-style-engine.php

```diff
@@ -549,18 +549,12 @@ function wp_style_engine_get_styles( $block_styles, $options = array() ) {
 	}
 	$defaults = array(
 		'selector'                   => null,
-		'context'                    => 'block-supports',
+		'context'                    => null,
 		'convert_vars_to_classnames' => false,
-		'enqueue'                    => false,
 	);
 
 	$options       = wp_parse_args( $options, $defaults );
-	$parsed_styles = null;
-
-	// Block supports styles.
-	if ( 'block-supports' === $options['context'] ) {
-		$parsed_styles = WP_Style_Engine::parse_block_styles( $block_styles, $options );
-	}
```

Comment:
Just making sure I'm following along — we no longer need the `block-supports` check, because this function effectively doesn't do anything if it isn't calling `WP_Style_Engine::parse_block_styles`?

Reply:
Good question! That answer is "yes, and..."

I reasoned that `wp_style_engine_get_styles` encounters only Gutenberg style objects, which are:

1. The value of `attributes.style` (block supports)
2. In the future
    i. the value of theme.json's root `styles` properties 
    ii. the value of theme.json's `styles.blocks[blockName]` properties 
    iii. the value of theme.json's `styles.elements[elementName]` properties 

These object share the same basic model, although there are some variations in property names and values. For example, global styles will eventually pass custom CSS properties such as `--wp--style--block-gap` or dynamic references like `{ "ref": "styles.color.text" }`.

The latter, I think can be dealt with internally when the time comes. 

Maybe. 

😇

Reply:
All sounds good to me — and removing the check means it encourages us to think holistically about `parse_block_styles` for the global styles context in addition to the block-level, so I think this is a good direction, too 👍